src/index.js

```diff
@@ -0,0 +1,117 @@
+const inquirer = require("inquirer");
+require("dotenv").config();
+
+const cTable = require("console.table");
+//  import questions
+const {
+  questions,
+  departmentQuestions,
+  employeeQuestions,
+  deleteQuestions,
+  askRoleQuestions,
+  getEmployeeDetails,
+  updateEmployees,
+} = require("./questions");
+
+// get the client
+const mysql = require("mysql2");
+const initDatabase = require("./db");
+
+const init = async () => {
+  try {
+    const { executeQuery, closeConnection } = await initDatabase({
+      host: process.env.DB_HOST,
+      user: process.env.DB_USER,
+      password: process.env.DB_PASSWORD,
+      database: process.env.DB_NAME,
+    });
+    let inProgress = true;
+
+    while (inProgress) {
+      // get answers for first set of questions
+      const answers = await inquirer.prompt(questions);
+
+      if (answers.proceed === "View all departments") {
+        const departments = await executeQuery("SELECT * FROM department");
+        console.table(departments);
+      } else if (answers.proceed === "view all roles") {
+        const roles = await executeQuery(
+          "SELECT  role.id, role.title, role.salary,department.departmentname FROM `role` INNER JOIN `department` ON role.department_id=department.id"
+        );
+        console.table(roles);
+      } else if (answers.proceed === "view all employees") {
+        const employees = await executeQuery(
+          "SELECT  e.id, e.first_name, e.last_name, CONCAT(m.first_name,' ',  m.last_name) AS manager, role.title, role.salary, department.departmentname    FROM employee AS e INNER JOIN `role` ON e.role_id=role.id LEFT JOIN `department` ON role.department_id = department.id LEFT JOIN `employee` AS m ON e.manager_id = m.id"
+        );
+        //     return results;
+        console.table(employees);
+      } else if (answers.proceed === "add a department") {
+        const departmentAnswers = await inquirer.prompt(departmentQuestions);
+        console.log(departmentAnswers);
+
+        const addDepartmentQuery = await executeQuery(
```

Comment:
This does not need to be declared as a variable because it is not used anywhere else, you are just displaying a console log to say that the department has been added. This is the same in the functions below.
```suggestion
        await executeQuery(
```